### PR TITLE
doc: Add run information to README instructions

### PR DESCRIPTION
**Pull request by secondl1ght**
The README was missing the run step after the build information. When I was setting up my local environment I found this missing step a bit confusing as to how to actually get the QML GUI running, so I think it would be good to include here.

image

### CODE REVIEW

**Review by shaavan — COMMENTED** [review]
Concept ACK

I tested that the added documentation is correct and works as intended.

Just one recommendation. It would be best to squash the commits together before they can be merged.
Reference: https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits

**Comment by secondl1ght** [comment]
Ok thanks for the review @shaavan, I have now squashed my commits into 1.

File: src/qml/README.md
Comment (hebasto) [inline src/qml/README.md:78]: Now it refers to an absolute path which is pretty questionable.

Usually, it's preferred to use paths that are relative to the repo's directory:
```
./src/qt/bitcoin-qt
```
or
```
src/qt/bitcoin-qt
```

Comment (secondl1ght) [reply src/qml/README.md:78]: The reason I chose not to use `./src/qt/bitcoin-qt` is because technically you could run the file from the src folder or also the qt folder directly by using `./bitcoin-qt`. However I can update it to whichever you prefer, for now I'll remove the initial forward slash to match your second option.

Comment (hebasto) [reply src/qml/README.md]: Technically, I doubt if someone will place a repository into the system root `/` directory. That is why an absolute path like `/src/qt/bitcoin-qt` is meaningless in sane scenarios.

Btw, `./src/qt/bitcoin-qt` and `src/qt/bitcoin-qt` are equivalent, no?

> ... technically you could run the file from the src folder or also the qt folder directly by using `./bitcoin-qt`.

Yes. If your goal is to document the easiest way that is one I use everyday dozens of times:
```
./autogen.sh
./configure
make
./src/qt/bitcoin-qt
```

In Bash when one starts a command with `./` auto-completion works nice for executables :)

I believe that readers of this file will understand that this way to run Bitcoin Core is on testing purpose only.

Comment (secondl1ght) [reply src/qml/README.md]: Ok sounds good, I think you have a good point which is that we want to document the most common and easiest way to run the file. I'll make the final change and then squash my commits again. Thanks!

File: src/qml/README.md
Comment (shaavan) [inline src/qml/README.md:76]: 2. To maintain consistency with line no. 68
`To add support for building the qml GUI,`
It would be best if you made the following change to the added message:
```suggestion
To run the qml GUI, execute the file:
```
It is better to keep terminology in our docs consistent, to begin with.

3. This one is a non-essential suggestion. How about mentioning, in the comment message itself, that the command should be run from root directory:
```suggestion
To run the qml GUI, run the following command from the root directory.
```

**Review by shaavan — COMMENTED** [review]
Changes since my last review:

- Squashed the commits together.
- `src/qt/bitcoin-qt` -> `./src/qt/bitcoin-qt`


Some minor nitpicks: 

1. The line `Remove forward slash from file path`, in the commit message does not seem to refer to any specific change done to this PR. I reckon it could be removed.

**Comment by secondl1ght** [comment]
Ok I have made the adjustments and I think that should be everything now, thanks for the reviews.

**Review by shaavan — APPROVED** [review]
ACK 7934def27e9b5746da266af9ec5ff09d73949ce0

Changes since my last review:
- Addressed [this](https://github.com/bitcoin-core/gui-qml/pull/90#pullrequestreview-836035548) comment.

I think this PR is complete and can be merged.

Btw, no need to mark me as a co-author :) Cheers :clinking_glasses:

File: src/qml/README.md
Comment (jarolrod) [inline src/qml/README.md:76]: I think the current text isn't optimal because:
1) Generally throughout our documentation, commands are written assuming that you are executing them from the root of the cloned repo. Including the text "... run the following command from the root directory" is redundant information, and actually a bit confusing (system root or repo root?)
2) The text has an awkward flow to it. We are saying "To run A, run B"

this really could just be the following, no more information is needed:
```suggestion
To run the qml GUI:
```

**Review by jarolrod — COMMENTED** [review]
Concept ACK

**Comment by secondl1ght** [comment]
Sounds good, I am not too attached to any specific wording, whatever the maintainers would like is fine with me. I just want there to be some kind of run command instructions in the README, I have merged the suggestion. Should I squash again?

**Comment by jarolrod** [comment]
@secondl1ght yes, you would need to squash again. We require a clean commit history, see: https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits

**Comment by secondl1ght** [comment]
Sounds good thanks, just squashed again.

**Comment by hebasto** [comment]
> Sounds good, I am not too attached to any specific wording, whatever the _maintainers would like_ is fine with me.

Not maintainers, but reviewers.

Maintainers follow reviewers' opinion.

**Review by hebasto — APPROVED** [review]
ACK 2e3f2a45fad6ecfda62bb4c182141f42184532bf

**Comment by jarolrod** [comment]
ACK 2e3f2a4

**Pull request merged**
